README.md

```diff
@@ -1,54 +1,210 @@
-# Sunrise symfony bundle installation guide
+# commercetools Symfony bundle
 
-## Requirements
-Symfony 3:
-To install symfony go to http://symfony.com/doc/current/book/installation.html 
+commercetools Symfony Bundle is collection of symfony bundles that ease the usage of 
+[commercetools PHP-SDK](https://github.com/commercetools/commercetools-php-sdk) when implementing
+a Symfony project.
+
+##### Table of contents
+
+  * [Requirements](#requirements)
+  * [Installation](#installation)
+    + [Verify everything works](#verify-everything-works)
+  * [Usage](#usage)
+    + [Usage of available services](#usage-of-available-services)
+      - [Available services](#available-services)
+    + [Console Commands](#console-commands)
+      - [Available console commands](#available-console-commands)
+    + [Usage of SDK Models in templates](#usage-of-sdk-models-in-templates)
+  * [Quick-start](#quick-start)
+  * [Disclaimer for ExampleBundle (dev)](#disclaimer-for-examplebundle--dev-)
+  * [Testing](#testing)
+  * [Issues](#issues)
+  * [Contribute](#contribute)
+  * [License](#license)
 
+
+## Requirements
 Composer :
 To install the composer go to https://getcomposer.org/doc/00-intro.md
 
 ## Installation
-Create a new or open a symfony3 project and open it in any editor of choice
+Create a new Symfony project 
+```sh
+composer create-project symfony/skeleton <project-name>
+```
+or use an existing one (symfony >= 3.4)
 
-Open composer.json and add to the attribute require this line
+Navigate to the project's directory and run
+```sh
+composer config extra.symfony.allow-contrib true
+```
+This will allow symfony to use the recipes-contrib (via Symfony Flex) that will automate
+most of the configuration.
 
+To install the commercetools symfony-bundle open composer.json and add to the attribute 
+`require` the package `"commercetools/symfony-bundle"`
+and run `composer install` or directly run the following on the command line
 ```sh
 composer require commercetools/symfony-bundle
 ```
 
```

Comment:
> Next, run the following from the command line 
> 
> ```sh
> composer install
> ```

nb. @nikossvnk you'll have to do this by hand

Reply:
I changed your suggestion a bit because the order of commands got mixed, I think